builder/builder_test.go

```diff
@@ -0,0 +1,44 @@
+package builder
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func mustExist(t *testing.T, name string) {
+	t.Helper()
+	_, err := os.Stat(name)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func cleanup(file string) {
+	_ = os.RemoveAll(file)
+}
+
+func TestNewDNFConfOnInit(t *testing.T) {
+	testDir, err := ioutil.TempDir("", "dnf-testdir-")
+	if err != nil {
+		t.Fatalf("couldn't create temporary directory to write test cases: %s", err)
+	}
+	defer cleanup(testDir)
+
+	b := New()
+	if err = b.Config.CreateCustomConfig(false, testDir); err != nil {
+		t.Errorf("Failed to create default config: %s", err)
+	}
+
+	if err = b.Config.LoadConfig(testDir + "/builder.conf"); err != nil {
+		t.Errorf("Failed to load default builder.conf: %s", err)
+	}
+	Offline = true
+
+	err = b.InitMix("25550", "10", false, false, true, "https://download.clearlinux.org", false)
```

Comment:
since this is an offline unit test can we make it more explicit that these are test values. so 25550 -> 10 and "https://download.clearlinux.org" -> "https://example.com"